api/src/pcapi/scripts/backoffice_users/add_permissions_to_staging_specific_roles.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from flask import Blueprint
+
+from pcapi.core.permissions.models import Permission
+from pcapi.core.permissions.models import Role
+from pcapi.core.permissions.models import RolePermission
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+blueprint = Blueprint("add_permissions_to_staging_specific_roles", __name__)
+
+
+def _purge_role_permissions(role: Role) -> None:
+    RolePermission.query.filter(RolePermission.roleId == role.id).delete()
+
+
+@blueprint.cli.command("add_permissions_to_staging_specific_roles")
+def add_permissions_to_staging_specific_roles() -> None:
+    """Add permissions to qa and global access roles"""
+    qa_role = Role.query.filter(Role.name == "qa").first()
+    global_access_role = Role.query.filter(Role.name == "global-access").first()
```

Comment:
(détail) Quand je lis ce code, je vois `.first()` et je me demande ce qu'on devrait faire s'il y a plusieurs éléments qui correspondent à ce critère. Mais en fait, ça n'est pas possible, car il y a une contrainte d'unicité sur `Role.name`. On pourrait éviter cette (petite) réflexion inutile en utilisant `one_or_none()`. Le résultat est le même, mais le code indique clairement qu'il ne peut y avoir que 0 ou 1 élément.

```suggestion
    qa_role = Role.query.filter(Role.name == "qa").one_or_none()
    global_access_role = Role.query.filter(Role.name == "global-access").one_or_none()
```